src/app/assets/languages/en.json

```diff
@@ -271,9 +271,13 @@
     },
     "cache-metrics":{
       "current-number-entries":"Current number of entries",
+      "current-number-entries-tooltip":"Displays the number of entries that are stored in the cache.",
       "min-nodes":"Minimum number of nodes",
+      "min-nodes-tooltip":"Displays the minimum number of nodes for the cache.",
```

Comment:
@danberindei What does "minimum number of nodes" reflect? How can we make the tooltip more explanatory?